src/App.tsx

```diff
@@ -1,61 +1,132 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+
+  const [errors, setErrors] = useState({
+    isTitleValid: true,
+    isUserSelected: true,
```

Comment:
```suggestion
    isTitleError: false,
    isUserError: false,
```

If your state is named `errors` so better to store error boolean  values here